wings/jwt.md

```diff
@@ -0,0 +1,61 @@
+# Wings API - JWT specifications
+
+This page lists the JWT specifications for Wings API.
+
+## Signing the token
+
+Make sure to sign the token using the `token` value from the node configuration.
+
+## Header format
+
+- Tokens typically expire in 5 minutes with few exceptions.
+- JWTs use the `HS256` algorithm.
+
+## Payload format
+
+**Make sure to include the other required values based on the route the JWT is for.**
+
+| Field      | Type     | Description                                                                                                            |
+| ---------- | -------- | ---------------------------------------------------------------------------------------------------------------------- |
+| iss        | string   | The value for `remote` in config.json. This is the panel's url.                                                        |
```

Comment:
Is this meant to be `config.yml`? I'm not sure where `config.json` is from.

Reply:
I meant config.yml. I'll fix the text later.

Reply:
Fixed!